server/utils/timezones.go

```diff
@@ -0,0 +1,610 @@
+package utils
```

Comment:
1. It's really hard to believe that this mapping does not yet exist in goland as it appears to be a common use case interacting with Microsoft software in general.
2. If it does not we 0/5 should work on making our implementation Google-able (separate repo? a submission to one of the MSGraph go clients?) in hope to make it used outside of mattermost.
3. Either way, suggest a sub-package under utils for the time being, `utils/tz`?

Reply:
https://golang.org/src/time/zoneinfo_abbrs_windows.go

The one resource I've found pertaining to using Windows timezones in the Go library is a non-exported map from the `time` package. Any use of the map seems to be internal to the library. The keys are Microsoft-compliant, and the values are abbreviations of timezones. These abbreviations are not compliant with `time.LoadLocation` across the board.

There is a comment to the right of each entry, showing an example IANA format for each Windows format. This is a much better resource, since it's directly from Go. Each of the timezones are compatible with `time`, as shown in the unit test I've included. I dismissed this originally as the information is in the comments, but we can parse the comments in the `make tz` script.